bin/webpack-entries.js

```diff
@@ -130,6 +130,8 @@ const entries = {
 		'add-to-cart-form':
 			'./assets/js/atomic/blocks/product-elements/add-to-cart-form/index.tsx',
 		...getBlockEntries( '{index,block,frontend}.{t,j}s{,x}' ),
+		'wc-blocks-classic-template-revert-button-style':
+			'./assets/js/templates/revert-button/style.scss',
```

Comment:
I noticed that all the values in other keys are referencing `.tsx`, `.ts`, `.js` or `.jsx` files, shouldn't we reference the `index.tsx` file in this case too?

```suggestion
		'wc-blocks-classic-template-revert-button-style':
			'./assets/js/templates/revert-button/index.tsx',
```

Reply:
Good call, updated for consistency with the other entries in this constant: [68e5415](https://github.com/woocommerce/woocommerce-blocks/pull/10388/commits/68e54150f47a7c2f82529102b6e9e3d17546f3f6)